### DISCUSSION TITLE
How to generate image/code block captions in markdown using tailwindcss typography

### DISCUSSION THREAD

**Discussion by alecgerona** [tailwindlabs/tailwindcss#5722] [Help] [answered]
So like the title, I saw in the demo of tailwincss that it does support image captioning. How does it do this? 

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/6406447/136308936-48bcd9be-1596-4170-8110-273ecffed1cc.png">

**Comment by adamwathan** [comment, marked as answer]
Hey! We use some `<figure>` markup which you can see when you view source:

![image](https://user-images.githubusercontent.com/4323180/136365043-02910b4c-469b-4548-b631-b7ca487b74b1.png)

```html
<figure>
  <img src="https://images.unsplash.com/photo-1556740758-90de374c12ad?ixlib=rb-1.2.1&amp;ixid=eyJhcHBfaWQiOjEyMDd9&amp;auto=format&amp;fit=crop&amp;w=1000&amp;q=80" alt="">
  <figcaption>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old.</figcaption>
</figure>
```

**Reply by alecgerona** [reply]
Thanks! I thought there's a specific markdown syntax you use to render it that way.

**Reply by alecgerona** [reply]
Can this figcaption also work on code blocks?
